### ISSUE TITLE
"Plugin Initialization Failed" error when trying to load in plugin

### ISSUE THREAD

**Issue by noah-guillory** [roxaskeyheart/Chromatics#7]
When trying to load Chromatics manually in ACT it gives me an error saying

"Unable to load DLL 'LogitechLedEnginesWrapper' : The specified module could not be found (Exception from HRESULT: 0x8007007E"

I have unblocked the .zip file before I extracted it to the plugins folder and made sure to have the Logitech Gaming Software open before opening ACT. I have even tried running ACT as administrator to see if it worked and it didn't. Am I doing something wrong?

The device that I am using is a Logitech G600

**Comment by logicallysynced** [comment]
In the Chromatics folder, open up the **lib** folder and locate the **LogitechLedEnginesWrapper.dll**. See if this is unblocked or not.

Additionally, Logitech Gaming Software needs to have the "Allow games to control illumination" checkbox selected.

**Comment by noah-guillory** [comment]
**LogitechLedEnginesWrapper.dll** doesn't show any options saying if blocked or not when I open it up in properties. I also have the "Allow games to control illumination" checkbox selected.

If it helps I am using Windows 10 x64

**Comment by logicallysynced** [comment]
Could you try installing the plugin from ACT? On the plugins tab there should be a Get Plugins button where you can select and install Chromatics. Please let me know if you still get this problem.

**Comment by dennym** [comment]
Hey there, got trouble with manually installing the plugin and tried it the in ACT way. There I run into "This assembly does not have a class that implements ACT's plugin interface"

And also afterwards I have Corale.Colore installed but not enabled. Enabling it results in the same error.

Adding it instead manually results in a
![errorpng](https://cloud.githubusercontent.com/assets/1562328/14227464/a85c5ca6-f8fe-11e5-99c4-29fa2a9f2103.PNG)

**Comment by logicallysynced** [comment]
When you download the ZIP file, Right click it and select Properties and then Unblock the file **before** extracting it. This should allow the assemblies to load when you attach the plugin to ACT.

Also please make sure you only add the Chromatics.dll to ACT.

Please open a new issue report if you still have issues so I can better help you.

**Comment by dennym** [comment]
Figured that out in the mean time. Its a bit tricky since you also have to unblock all the files in the lib folder. I guess thats why the install via ACT itself does not not work, since these files are not unblocked at first.

Thanks anyways. For all others who got here. Unblock the files. I guess its also the solution for @slokomisu
